assets/xml/objects/object_icicle.xml

```diff
@@ -1,7 +1,8 @@
 ﻿<Root>
+    <!-- Objects for Icicles: The breakable ice stalagmites sticking out of the ground in Snowhead Temple -->
```

Comment:
This comment is incorrect. This object is used for both the one on the floor and the one on the ceiling. It's pretty easy to check in-game; just change the asset and look in the very first room in Snowhead Temple:
![image](https://github.com/zeldaret/mm/assets/12245827/2a5add0f-5ef5-4acb-bbb7-ba51500a00f3)

I know you added this because of a comment on this PR, but I think the best thing to do here would be to delete everything after the colon in here; it's better to be vague than incorrect.

Reply:
Ah yeah that was my bad, should have double checked